src/react/useWindup.ts

```diff
@@ -100,6 +106,36 @@ export default function useWindup<M extends HookMetadata>(
     }
   }, [windupIsFinished]);
 
+  const pause = React.useCallback(() => {
+    if (isPausedRef.current === true) {
+      return;
+    }
+
+    if (timeoutRef.current) {
+      isPausedRef.current = true;
+
+      clearTimeout(timeoutRef.current);
+      pauseDelayRemainingRef.current = Math.max(
+        0,
+        nextCharAtRef.current ?? 0 - Date.now()
+      );
+    }
+  }, [isPausedRef, timeoutRef, nextCharAtRef, pauseDelayRemainingRef]);
```

Comment:
You can safely leave the refs out of the dependency array.

Reply:
Oop, good point! Will do